src/components/DataElement/CitiesSplit.jsx

```diff
@@ -20,19 +23,31 @@ export default function CitiesSplit({ isLoading, data }) {
       >
         {isLoading && <Loader />}
         {data &&
-          <ResponsiveContainer height={data.length * 32} width={'99%'}>
-            <BarChart
-              data={data}
-              layout="vertical"
-            >
-              <YAxis dataKey="city" type="category" tick={{ fill: theme.colors.contentPrimary }} width={100} />
-              <XAxis type="number" hide />
-              <Tooltip
-                formatter={value => [value, 'Liczba']}
-              />
-              <Bar dataKey="count" fill={theme.colors.accent} />
-            </BarChart>
-          </ResponsiveContainer>}
+          <DataContext.Consumer>{({ setPosition, setZoom }) => (
+            <ResponsiveContainer height={data.length * 32} width={'99%'}>
+              <BarChart
+                data={data}
+                layout="vertical"
+              >
+                <YAxis
+                  dataKey="city"
+                  type="category"
+                  tick={{ fill: theme.colors.contentPrimary }}
+                  width={100}
+                  onClick={({ value }) => {
+                    setPosition(find(data, { city: value }).location);
```

Comment:
I don't think we have to use `find` method of lodash. Doesn't `filter` method returns the same result?

Reply:
Done